plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,22 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works on D3D. OpenGL has no such issues.
 				if(fi.FBMSK == 0x00000)
 				{
-					skip = 28; // outline
-				}
-				if(fi.FBMSK == 0x00FFFFFF)
-				{
-					skip = 1;
+					// Hack removes character outlines as well as black borders on the sides of the screen.
+					// In the past the hack was used to remove red vertical lines on the screen I believe but now that issue has been fixed.
```

Comment:
You're right , skip 1 did that , I wasn't able to find any render problems with the outlines on 1.4 and compiling 1.2 seems a bit of a pain since it's for vs 13

Honestly I have no idea why outlines were removed unless they caused some specific issues pre 1.4